admin/src/components/CreationFormular.vue

```diff
@@ -190,6 +190,7 @@ export default {
     // Auswählen eines Zeitraumes
     updateRadioSelected(name, index, openCreation) {
       this.createdIndex = index
+      this.text = 'Schöpfung für ' + name.short + ' ' + this.$moment().format('YYYY')
```

Comment:
We make a message for the CreationFormular why do we write only the year? 
```
this.$moment().format('YYYY')
```

Reply:
Beispiel: 
name.short = November
this.$moment().format('YYYY') = 2021 

this.text = 'Schöpfung für ' + name.short + ' ' + this.$moment().format('YYYY')

this.text = 'Schöpfung für November 2021